### PR TITLE
More background on Siri female voice recognition issue

### PR DESCRIPTION
**Pull request by macdonst**
Hey, please feel free to ignore this PR as it is probably too wordy but I thought you may be interested in a bit of the background. 

I used to work in Speech Recognition and I still do presentations on it from time to time and the issue where recognition rates for women are low has been a problem since day one. Imagine a bunch of engineers, probably all men, were working on a way of recognizing speech; they created their algorithms and started training it with data they selected. 

Of course if those engineers couldn't recognize the speech easily they did not feed those utterances into their engine. The problem is most of these engineers were men and likely were suffering from various levels of high-frequency hearing loss. 

Men and women have different pitched voices because of the length and thickness of their vocal cords. This leads to women having higher pitched voices, typically. 

Engineers included an unconscious bias by excluding utterances they could not hear well which means less utterances by females were not included in the training data. So from the very beginning speech recognition engines were improperly tuned and had a hard time recognizing the female voice.

I use this example when I'm talking to people to implore them to hire diverse teams and don't get stuck on a one gender, age, nationality, etc.

Anyway, I thought you might be interested in the history of all this. Feel free to ignore me, I can talk about speech rec all day.

### CODE REVIEW

**Comment by jessynd** [comment]
Thank you so much for this!

**Comment by macdonst** [comment]
No problem, you didn't have to merge it. I just wanted to give some more background. Halfway through the PR I was questioning whether or not I was a mansplaining idiot or not.
